docusaurus/android_versioned_docs/version-5.x.x/04-compose/03-general-customization/01-chat-theme.mdx

```diff
@@ -101,128 +99,128 @@ Let's see what each property exposes and what the values are used for.
 
 ### StreamDimens
 
 `StreamDimens` defines different sizes that can be customized in the SDK.
 
-You can find the definitions of all the dimensions we expose in the [class documentation](https://github.com/GetStream/stream-chat-android/blob/main/stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/theme/StreamDimens.kt), as well as what the default dimensions are.
+You can find the definitions of all the dimensions we expose in the [class documentation](https://github.com/GetStream/stream-chat-android/blob/v5/stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/theme/StreamDimens.kt), as well as what the default dimensions are.
 
 You can also browse which components are using the dimensions, to know what will be affected by any change.
 
 ### StreamTypography
 
 `StreamTypography` is used to apply different font weights and sizes to our textual UI components.
 
-You can find all the text style properties we expose in the [class documentation](https://github.com/GetStream/stream-chat-android/blob/main/stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/theme/StreamTypography.kt), as well as what the default styles are.
+You can find all the text style properties we expose in the [class documentation](https://github.com/GetStream/stream-chat-android/blob/v5/stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/theme/StreamTypography.kt), as well as what the default styles are.
 
 You can also browse which components are using the styles, to know what will be affected by any change.
 
 ### StreamShapes
 
 `StreamShapes` provides a small collection of shapes that let us style our containers.
 
-You can find all the shapes we expose in the [class documentation](https://github.com/GetStream/stream-chat-android/blob/main/stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/theme/StreamShapes.kt), as well as what the default shapes are.
+You can find all the shapes we expose in the [class documentation](https://github.com/GetStream/stream-chat-android/blob/v5/stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/theme/StreamShapes.kt), as well as what the default shapes are.
 
 These are really easy to customize, as you've seen before, and can make your app feel closer to your design system.
 
 ### RippleTheme
 
 Defines the appearance for ripples. The default ripple theme is `StreamRippleTheme`.
 
-You can find out more about it by reading the [class documentation](https://github.com/GetStream/stream-chat-android/blob/main/stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/utils/StreamRippleTheme.kt).
+You can find out more about it by reading the [object documentation](https://github.com/GetStream/stream-chat-android/blob/v5/stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/theme/StreamRippleTheme.kt).
```

Comment:
Stating object documentation feels a bit weird, but it's not a class so that feels weird as well.